atlas/visfAtlas/README.rtf

```diff
@@ -0,0 +1,114 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1671\cocoasubrtf600
+{\fonttbl\f0\froman\fcharset0 TimesNewRomanPSMT;\f1\froman\fcharset0 TimesNewRomanPS-BoldMT;}
+{\colortbl;\red255\green255\blue255;\red42\green44\blue46;\red191\green191\blue191;\red231\green234\blue235;
```

Comment:
think you can try pandoc to convert this to markdown

https://pandoc.org/installing.html#macos

```
pandoc -s README.rtf -o README.md
```

Reply:
did not use pandoc cause it is taking ages to install, LMK if the output is ok